hwy/ops/generic_ops-inl.h

```diff
@@ -1249,6 +1249,35 @@ HWY_API V PopulationCount(V v) {
 }
 #endif
 
+HWY_API Vec128<int64_t,1> operator*(Vec128<int64_t,1> x, Vec128<int64_t,1> y) {
+    return Set(Simd<int64_t,1,0>(), GetLane(x)*GetLane(y));
+}
+HWY_API Vec128<uint64_t,1> operator*(Vec128<uint64_t,1> x, Vec128<uint64_t,1> y) {
+    return Set(Simd<uint64_t,1,0>(), GetLane(x)*GetLane(y));
+}
+
+#ifndef HWY_NATIVE_I64MULLO
+
+template <class V, typename T = LaneType<V>, HWY_IF_LANE_SIZE(T, 8), HWY_IF_UNSIGNED(T)>
+HWY_API V operator*(V x, V y) {
+  DFromV<V> d64;
+  RepartitionToNarrow<decltype(d64)> d32;
+  auto x32 = BitCast(d32, x);
+  auto y32 = BitCast(d32, y);
+  V lolo = MulEven(x32, y32);
+  auto lohi = BitCast(d32, MulEven(x32, BitCast(d32, ShiftRight<32>(y))));
+  auto hilo = BitCast(d32, MulEven(BitCast(d32, ShiftRight<32>(x)), y32));
+  return lolo + ShiftLeft<32>(BitCast(d64, lohi + hilo));
+}
+template <class V, typename T = LaneType<V>, HWY_IF_LANE_SIZE(T, 8), HWY_IF_SIGNED(T)>
+HWY_API V operator*(V x, V y) {
+  DFromV<V> di64;
+  Repartition<uint64_t, decltype(di64)> du64;
```

Comment:
Minor: Repartition is intended for changes of lane size, in this case we want the same number of lanes, so you can use `RebindToUnsigned<decltype(di64)> du64`.